parm/config/config.wave

```diff
@@ -93,9 +91,16 @@ if [ "${CDUMP}" != gfs ]; then    # Setting is valid for GDAS and GEFS
 else                              # This is a GFS run
   rst_dt_gfs=$(( restart_interval_gfs * 3600 ))
   export RSTTYPE_WAV='F'                 # generate second tier of restart files
-  if [ $rst_dt_gfs -gt 0 ]; then export RSTTYPE_WAV='T' ; fi
-  export DT_1_RST_WAV=${rst_dt_gfs:-0}   # time between restart files, set to DTRST=1 for a single restart file
-  export DT_2_RST_WAV=${rst_dt_gfs:-0}   # restart stride for checkpointing restart
+  if [ $rst_dt_gfs -gt 0 ]; then 
+    export RSTTYPE_WAV='T' 
```

Comment:
This is in both branches of the if, so it should be placed before the branch instead of L93